ctapipe/instrument/camera/geometry.py

```diff
@@ -88,25 +88,32 @@ class CameraGeometry:
 
     Parameters
     ----------
-    self : type
-        description
     name : str
-         Camera name (e.g. NectarCam, LSTCam, ...)
-    pix_id : array(int)
+         Camera name (e.g. "NectarCam", "LSTCam", ...)
+    pix_id : np.ndarray[int]
         pixels id numbers
-    pix_x : array with units
+    pix_x : u.Quantity
         position of each pixel (x-coordinate)
-    pix_y : array with units
+    pix_y : u.Quantity
         position of each pixel (y-coordinate)
-    pix_area : array(float)
-        surface area of each pixel, if None will be calculated
```

Comment:
was this "if not given will be calculated" simply not correct?

Reply:
Yes, this got removed a long time ago and made a required argument